client/verta/tests/test_endpoint/test_endpoint.py

```diff
@@ -683,3 +683,34 @@ def test_update_from_version_diff_workspace_no_access_error(self, client_2, mode
         excinfo_value = str(excinfo.value).strip()
         assert "403" in excinfo_value
         assert "Access Denied" in excinfo_value
+
+    def test_predict_dataframe_input(self, client, created_endpoints, model_version):
+        np = pytest.importorskip("numpy")
+        sklearn = pytest.importorskip("sklearn")
+        pd = pytest.importorskip("pandas")
+
+        endpoint = client.create_endpoint(_utils.generate_default_name())
+        created_endpoints.append(endpoint)
+
+        input = pd.DataFrame({"feature-1": [45], "feature-2": [43]})
+
+        with sys_path_manager() as sys_path:
+            sys_path.append(".")
+
+            from dataframe_custom_model.dataframe_model import DataFrameModel  # pylint: disable=import-error
+
+            df_model = DataFrameModel()
+            model_version.log_model(df_model, custom_modules=[
+                "dataframe_custom_model/",
+                "../verta/"
```

Comment:
I think this is required so that the `prediction_input_df` decorator is found.

Reply:
It only looks like it's required because we haven't published a Client version with that decorator.

I'd recommend

1. Increment [the version number](https://github.com/VertaAI/modeldb/blob/master/client/verta/verta/__about__.py#L9) to `0.15.9a0`
2. Run our `client/publish-pypi` job, passing in this branch for `client_branch`
3. Delete this line
4. Run this test

This is important to validate, because we shouldn't need to send over the Client as a custom module for this functionality.

Reply:
Do I need to include `verta==0.15.9a0` in the requirements instead?

Reply:
Nvm, it works without it. Probably pick up the verta version information locally.

Reply:
It picks up the version the user has installed installed locally 👍